blocks/rss-fbia-feature-block/features/rss/xml.js

```diff
@@ -174,10 +174,17 @@ export function FbiaRss({ globalContent, customFields, arcSite }) {
     adDensity,
     placementSection,
     adScripts,
+    customMeta
   ) {
     BuildContent.call(this)
 
-    this.buildHTMLHead = (s, domain, resizerWidth, resizerHeight) => {
+    this.buildHTMLHead = (s, domain) => {
+      customMetaXml = customMeta.forEach(cf =>{
```

Comment:
Missing const

You have customMeta defined as a string at L636 so you can't forEach customMeta

I was thinking of using a kvp instead. It needs to be able to handle multiple meta tags, and each meta tag has two attributes property and content.  If you try to setup two meta tags in the kvp like 
![Screen Shot 2021-03-05 at 12 20 29 PM](https://user-images.githubusercontent.com/7131596/110150488-5394b180-7dad-11eb-806a-6cf6ab84fd47.png)
they will overwrite each other and you will only have one set.

So a kvp will not work.

xmlbuilder2 can parse xml also.  I recommend you keep customMeta a string and have them enter the tags like ` <meta property="foo" content="bar"><meta property="quii" content="baz">` then you can convert that to an object like ```x = xmlbuilder.create(customMeta)
metaObject = x.end({format: 'object'})```